config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml

```diff
@@ -86,7 +93,6 @@ periodics:
       - --gcp-node-image=gci
       - --gcp-nodes=2
       - --gcp-project=k8s-infra-e2e-gpu-project
-      - --gcp-zone=us-west1-b
```

Comment:
we don't need the zone?

Reply:
No. Boskos will default to `us-central1`. This canary will validate this assumption.